src/main/resources/Jaina/localization/ENG/cards.json

```diff
@@ -92,50 +92,50 @@
   },
   "jaina:Counterspell": {
     "NAME": "Counter Spell",
-    "DESCRIPTION": "Choose one: NL Gain 1 Intangible, 1 jaina:Buffer or 1 Artifact. NL Ethereal.",
+    "DESCRIPTION": "Ethereal. NL Choose one: NL Gain 1 Intangible, 1 jaina:Buffer or 1 Artifact. NL Exhaust.",
```

Comment:
since Counterspell is a POWER, I should remove 'Exhaust' in descriptions.

Reply:
Sure, go ahead